src/daft-dsl/src/functions/registry.rs

```diff
@@ -0,0 +1,33 @@
+use std::{collections::HashMap, sync::Arc};
+
+use super::{hash::HashFunction, ScalarUDF};
+
+lazy_static::lazy_static! {
+    pub static ref REGISTRY: Registry = Registry::new();
```

Comment:
Should we make this threadsafe? We have had issues before with Ray starting up importing daft in multiple threads.